keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/CheckScheduler.kt

```diff
@@ -96,16 +100,12 @@ class CheckScheduler(
         }
       }
       runBlocking { job.join() }
-      log.debug("Scheduled resource validation complete")
```

Comment:
The value (such as it is) of these logs is in just knowing the loops didn't get stuck. We'll be able to see that from telemetry, right?

Reply:
Yep, I added a duration metric, so if we see that increasing we will know that something is funky. Or at least, that is my theory.